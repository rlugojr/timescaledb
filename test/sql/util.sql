\unset ECHO
\o /dev/null
\ir include/create_single_db.sql
\o
\set ECHO errors
\set VERBOSITY default

DO $$
BEGIN
  ASSERT( _timescaledb_catalog.get_partition_for_key('', 16:: INT4) = 13 );
  ASSERT( _timescaledb_catalog.get_partition_for_key('dev1', 16:: INT4) = 4 );
  ASSERT( _timescaledb_catalog.get_partition_for_key('longlonglonglongpartitionkey', 16:: INT4) = 6 );
END$$;
