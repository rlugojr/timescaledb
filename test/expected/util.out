\unset ECHO
psql:include/create_single_db.sql:7: NOTICE:  installing required extension "postgres_fdw"
